src/index.html

```diff
@@ -4,9 +4,280 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Handjet:wght@200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover">
+              Laptops & computers
+            </a>
```

Comment:
```suggestion
            <a
              class="nav__link"
              href="#"
              data-qa="nav-hover"
            >
              Laptops & computers
            </a>
```
Keep your attributes correctly formatted